src/components/shared/AlertCard/AlertCard.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+import AlertCardContainer from './AlertCard.container';
+import Card from 'shared/Card';
+import Svg from 'shared/Svg';
+import Button from 'shared/Button';
+
+type Props = Partial<{
+  buttonBackgroundColor: string;
+  cta: string;
+  ctaColor: string;
+  message: string;
+  onClose: () => void;
+  src: string;
+  srcColor: string;
+  title: string;
+}>
+
+const AlertCard = ({ buttonBackgroundColor, cta, ctaColor, message, onClose, src, srcColor, title }: Props) => {
+  return (
+    <Card
```

Comment:
I think we can get rid of the height/width/padding properties. We can move the `AlertCardContainer` on the top level and then target the `Card` class name directly and add the height/width/padding there

Reply:
will also need height and width 100% on the card itself

Reply:
you can also have a default padding and overwrite it on the class

Reply:
updated